scripts/process-releases.js

```diff
@@ -66,8 +66,13 @@ async function processReleases(opts) {
     // ### Customization
     await addMetadataToFile(join(docDestination, 'index.md'), {
       title: 'Introduction',
+      ['sidebar_position']: 0,
       [`displayed_sidebar`]: 'docsSidebar',
     })
+    await addMetadataToFile(join(docDestination, 'Guides', 'Getting-Started.md'), { ['sidebar_position']: 0 })
+    await addMetadataToFile(join(docDestination, 'Guides', 'Database.md'), { ['sidebar_position']: 1 })
+    await addMetadataToFile(join(docDestination, 'Guides', 'Recommendations.md'), { ['sidebar_position']: 2 })
+    await addMetadataToFile(join(docDestination, 'Guides', 'Testing.md'), { ['sidebar_position']: 3 })
```

Comment:
This will retroactively set the sidebar position for old versions. If the page does not exist it is a no-op.

Reply:
Perfect!